### PR TITLE
Fix collapsible example #trivial

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Add eCommerce related example instead of weak text.

#### What problem is this solving?
part of https://github.com/vtex/styleguide/issues/1149 and https://github.com/vtex/styleguide/issues/1124

#### How should this be manually tested?
`yarn styleguide`

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Collapsible/README.md [hunk 1]
```diff
@@ -229,63 +257,97 @@ function toggleAccordion(questionNbr) {
...
         </Collapsible>
 
-        <hr className="ma0 bb bb-0 b--black-10"/>
+        <hr className="ma0 bb bb-0 b--black-10" />
 
         <Collapsible
           header={
             <div className="pv6 hover-c-on-action-secondary">
-              What is your favorite color?
+              What is the PCI SSC
```
Comment (lucasecdb) [inline react/components/Collapsible/README.md:306]:
```suggestion
              What is the PCI SSC?
```

Comment (kevinch) [reply react/components/Collapsible/README.md]: Good catch.

File: react/components/Collapsible/README.md [hunk 2]
```diff
@@ -49,16 +31,15 @@ initialState = { isOpen: false }
...
-      isOpen={state.isOpen}
-      >
-        <div className="mt4">
-          <div className="mb3">
-            <Input label="Surname" />
-          </div>
-          <div className="mb3">
-            <Input label="Nickname" />
-          </div>
+      isOpen={state.isOpen}>
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:34]:
```suggestion
      isOpen={state.isOpen}
    >
```

File: react/components/Collapsible/README.md [hunk 3]
```diff
@@ -69,35 +50,30 @@ Custom Caret Color Example
...
-      <div className="mt4">
-        Here goes your content
-      </div>
-  </Collapsible>
+  <div className="mb5">
+    <Collapsible
+      header={<span>Here goes your base header</span>}
+      onClick={e => setState({ isOpen1: e.target.isOpen })}
+      isOpen={state.isOpen1}
+      caretColor="base">
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:58]:
```suggestion
      caretColor="base"
    >
```

File: react/components/Collapsible/README.md [hunk 4]
```diff
@@ -107,103 +83,154 @@ Navigation bar example
...
-      </div>
+        header={
+          <div className="c-on-base hover-c-emphasis">
+            <span>
+              <svg
+                width="16"
+                height="16"
+                viewBox="0 0 16 16"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg">
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:98]:
```suggestion
                xmlns="http://www.w3.org/2000/svg"
              >
```

File: react/components/Collapsible/README.md [hunk 5]
```diff
@@ -107,103 +83,154 @@ Navigation bar example
...
+                />
+              </svg>
+            </span>
+            <span className="ml5 fw5">Catalog</span>
+          </div>
+        }
+        align="right"
+        onClick={e => setState({ isOpen1: e.target.isOpen })}
+        isOpen={state.isOpen1}
+        caretColor="muted">
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:119]:
```suggestion
        caretColor="muted"
      >
```

File: react/components/Collapsible/README.md [hunk 6]
```diff
@@ -107,103 +83,154 @@ Navigation bar example
...
+      <Collapsible
+        header={
+          <div className="c-on-base hover-c-emphasis">
+            <span>
+              <svg
+                width="16"
+                height="16"
+                viewBox="0 0 16 16"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg">
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:150]:
```suggestion
                xmlns="http://www.w3.org/2000/svg"
              >
```

File: react/components/Collapsible/README.md [hunk 7]
```diff
@@ -107,103 +83,154 @@ Navigation bar example
...
+                />
+              </svg>
+            </span>
+            <span className="ml5 fw5">Payments</span>
+          </div>
+        }
+        align="right"
+        onClick={e => setState({ isOpen2: e.target.isOpen })}
+        isOpen={state.isOpen2}
+        caretColor="muted">
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:169]:
```suggestion
        caretColor="muted"
      >
```

File: react/components/Collapsible/README.md [hunk 8]
```diff
@@ -107,103 +83,154 @@ Navigation bar example
...
+      <Collapsible
+        header={
+          <div className="c-on-base hover-c-emphasis">
+            <span>
+              <svg
+                width="16"
+                height="16"
+                viewBox="0 0 16 16"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg">
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:197]:
```suggestion
                xmlns="http://www.w3.org/2000/svg"
              >
```

File: react/components/Collapsible/README.md [hunk 9]
```diff
@@ -107,103 +83,154 @@ Navigation bar example
...
-        </Collapsible>
-      </div>
+            </span>
+            <span className="ml5 fw5">Inventory & shipping</span>
+          </div>
+        }
+        align="right"
+        onClick={e => setState({ isOpen3: e.target.isOpen })}
+        isOpen={state.isOpen3}
+        caretColor="muted">
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:214]:
```suggestion
        caretColor="muted"
      >
```

File: react/components/Collapsible/README.md [hunk 10]
```diff
@@ -229,63 +257,97 @@ function toggleAccordion(questionNbr) {
...
+              What is a sub-acquirer?
             </div>
           }
           onClick={toggleAccordion(1)}
-          isOpen={ state.openQuestion===1 }
-          >
-            <div className="bg-muted-5 pa6">
-              Arthur, King of the Britons.
-            </div>
+          isOpen={state.openQuestion === 1}>
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:264]:
```suggestion
          isOpen={state.openQuestion === 1}
        >
```

File: react/components/Collapsible/README.md [hunk 11]
```diff
@@ -229,63 +257,97 @@ function toggleAccordion(questionNbr) {
...
+              What is the connector?
             </div>
           }
           onClick={toggleAccordion(2)}
-          isOpen={ state.openQuestion===2 }
-          >
-            <div className="bg-muted-5 pa6">
-              To seek the Holy Grail.
-            </div>
+          isOpen={state.openQuestion === 2}>
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:286]:
```suggestion
          isOpen={state.openQuestion === 2}
        >
```

File: react/components/Collapsible/README.md [hunk 12]
```diff
@@ -229,63 +257,97 @@ function toggleAccordion(questionNbr) {
...
+              What is the PCI SSC?
             </div>
           }
           onClick={toggleAccordion(3)}
-          isOpen={ state.openQuestion===3 }
-          >
-            <div className="bg-muted-5 pa6">
-              Pink.
-            </div>
+          isOpen={state.openQuestion === 3}>
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:310]:
```suggestion
          isOpen={state.openQuestion === 3}
        >
```

File: react/components/Collapsible/README.md [hunk 13]
```diff
@@ -229,63 +257,97 @@ function toggleAccordion(questionNbr) {
...
+              What is a transaction?
             </div>
           }
           onClick={toggleAccordion(4)}
-          isOpen={ state.openQuestion===4 }
-          >
-            <div className="bg-muted-5 pa6">
-              What do you mean? An African or European swallow?
-            </div>
+          isOpen={state.openQuestion === 4}>
```
Comment (emersonlaurentino) [inline react/components/Collapsible/README.md:333]:
```suggestion
          isOpen={state.openQuestion === 4}
        >
```

File: react/components/Collapsible/README.md (hunk 2 above)
Comment (kevinch) [reply react/components/Collapsible/README.md:34]: Strange, why wouldn't my Prettier do this by itself?

Comment (emersonlaurentino) [reply react/components/Collapsible/README.md:34]: because it doesn't look at the .md files

**Pull request merged**